### PR TITLE
Fix/disabled button css

### PR DESCRIPTION
**Pull request by richard-ramos**

### CODE REVIEW

File: src/index.scss [hunk 1]
```diff
@@ -31,7 +31,12 @@ body {
 
   .clickable {
     cursor: pointer;
+    
+    .disabled {
```
Comment (jrainville) [inline src/index.scss]: Are you sure you don't mean `&.disabled`?

Comment (richard-ramos) [reply src/index.scss]: you're right!

Comment (richard-ramos) [reply src/index.scss]: ah no. the disabled is applied to childs of .clickable :)

Comment (jrainville) [reply src/index.scss]: Ah ok, I trust you :)

**Pull request merged**